### PR TITLE
Grammar Patch

### PR DESCRIPTION
**Pull request by VGrol**
I took the liberty to fix the grammar echo outputs.

### CODE REVIEW

File: src/hooks/ykfde [hunk 1]
```diff
@@ -82,28 +82,28 @@ ykfde_do_it() {
 
     if [ -z "$_passphrase" ]; then
         if [ -n "$YKFDE_CHALLENGE" ]; then
-          printf " > Failed to do challenge-response.\n   Falling back to manual passphrase entering to unlock the disk.\n"
-          [ "$trial_nr" -le "$YKFDE_CRYPTSETUP_TRIALS" ] && echo "   Just press ENTER to skip and to try Challenge-Response again."
+          printf " > Challenge-response failed.\n   Falling back to manual passphrase.\n"
+          [ "$trial_nr" -le "$YKFDE_CRYPTSETUP_TRIALS" ] && echo "   Press ENTER to skip and retry Challenge-Response."
         else
-          echo " > We need the passphrase to unlock the disk."
+          echo " > Passphrase to needed to unlock disk."
```
Comment (Vincent43) [inline src/hooks/ykfde]: Is this valid sentence?

File: src/ykfde-enroll [hunk 2]
```diff
@@ -77,24 +77,23 @@ fi
...
-[ "$DBG" ] && printf "\n   Got as response: '$_passphrase'\n"
+[ "$DBG" ] && printf "\n   Received response: '$_passphrase'\n"
 
 if [ -n "$_passphrase" ] && [ "$YKFDE_CHALLENGE_PASSWORD_NEEDED" = "1" ]; then
     _passphrase="$_pw$_passphrase"
 fi
 
 [ "$DBG" ] && echo " > Using '$_passphrase' with cryptsetup!"
-echo "You may now be prompted for an existing passphrase. This is NOT the passphrase you just entered, this is the passphrase that you currently use to unlock your LUKS encrypted drive."
-printf "   Enter password: "; if [ "$DBG" ]; then read -r OLD; else read -r -s OLD; fi
```
Comment (Vincent43) [inline src/ykfde-enroll]: You removed password prompt :smile:

File: src/ykfde-enroll [hunk 3]
```diff
@@ -77,24 +77,23 @@ fi
...
+echo "If prompted for a passphrase, enter the old registered passphrase for your LUKS container."
```
Comment (Vincent43) [inline src/ykfde-enroll]: You removed password prompt :smile:

Comment (VGrol) [reply src/ykfde-enroll]: Yeah, I was conflicted on that change.
initially I figured the stand-alone prompt would be enough, but I only learned later on that the script relied mostly on the combined prompt.

File: src/hooks/ykfde (hunk 1 above)
Comment (VGrol) [reply src/hooks/ykfde]: Good catch, I missed that one myself.

**Comment by VGrol** [comment]
Requested changes were implemented.

File: src/ykfde-enroll [hunk 4]
```diff
@@ -77,24 +77,23 @@ fi
...
-[ "$DBG" ] && printf "\n   Got as response: '$_passphrase'\n"
+[ "$DBG" ] && printf "\n   Received response: '$_passphrase'\n"
 
 if [ -n "$_passphrase" ] && [ "$YKFDE_CHALLENGE_PASSWORD_NEEDED" = "1" ]; then
     _passphrase="$_pw$_passphrase"
 fi
 
 [ "$DBG" ] && echo " > Using '$_passphrase' with cryptsetup!"
-echo "You may now be prompted for an existing passphrase. This is NOT the passphrase you just entered, this is the passphrase that you currently use to unlock your LUKS encrypted drive."
-printf "   Enter password: "; if [ "$DBG" ]; then read -r OLD; else read -r -s OLD; fi
```
Comment (Vincent43) [inline src/ykfde-enroll]: Probably you misunderstood me. The second line you deleted is the password prompt. Without it the script will be broken. Please restore this line

**Comment by Vincent43** [comment]
Ok, I adjusted it myself.

**Pull request merged**